### DISCUSSION TITLE
Clarification on Transaction Batching in Optimism

### DISCUSSION THREAD

**Discussion by doukhahmed** [ethereum-optimism/developers#533] [General]
### Discussion Topic

Hello Optimism Community,

As a developer, do I need to handle the process of batching transactions and submitting them to Layer 1, or is this fully managed by the Optimism protocol?

**Comment by imtipi** [comment]
if you are talking about op mainnet,no.
just use like the normal evm network,sequencer and batcher will handle these

**Reply by doukhahmed** [reply]
for instance, i use sepolia.

I understand that the process of batching transactions and submitting them to Layer 1 (Ethereum) is managed by the infrastructure of Optimism, involving operators or validators. 

So for me it's like an abstract process ?

**Reply by imtipi** [reply]
yes,you only need running your own sequencer and batcher if you want to run your own OP Stack chain.
use the network doesn't need to handle these

**Reply by doukhahmed** [reply]
Can i run my own OP Stack chain without submitting txs to layer 1 ?

**Reply by imtipi** [reply]
you can't,although you can try using other evm network as L1(didn't tested yet),or post batch data to [Alt-da](https://docs.optimism.io/builders/chain-operators/features/alt-da-mode)(Celestia EigenDA Avail) to lower your running cost
